packages/database/src/sync/buildEncounterLinkedLookupFilter.ts

```diff
@@ -1,12 +1,55 @@
-import { buildEncounterLinkedSyncFilterJoins } from './buildEncounterLinkedSyncFilter';
+import {
+  buildEncounterLinkedSyncFilterJoins,
+  type JoinConfig,
+} from './buildEncounterLinkedSyncFilter';
 import { buildSyncLookupSelect } from './buildSyncLookupSelect';
 import type { Model } from '../models/Model';
 
+export type Options = {
+  extraSelects?: Record<string, string>; // extra selects to attach to the lookup select
+  extraJoins?: JoinConfig[]; // extra joins needed to traverse between this model and the encounters table
```

Comment:
```suggestion
  joinsToEncounters?: JoinConfig[]; // extra joins needed to traverse between this model and the encounters table
```
Matching naming in params below

Reply:
Oh yo is this used??

Reply:
oh it isnt!